### ISSUE TITLE
IP config

### ISSUE THREAD

**Issue by B-u-R-n-I-n-G-f-I-r-E** [gwojcieszczuk/kms4vsphere#4]
What do i need to do to setup the kms with a static network config?

**Comment by gwojcieszczuk** [comment]
Will come up with instructions hopefully this evening, and will post it in project description.

**Comment by B-u-R-n-I-n-G-f-I-r-E** [comment]
Is there any update?

**Comment by gwojcieszczuk** [comment]
Just updated README file on this github page. Check instructions there.

**Comment by B-u-R-n-I-n-G-f-I-r-E** [comment]
Okay, that works almost.
After that change i can not access the web page anymore to get the certificates.

**Comment by gwojcieszczuk** [comment]
Yup, forgot about firewall DNAT rule. Updated instructions in README page.

**Comment by B-u-R-n-I-n-G-f-I-r-E** [comment]
This was partially working.
After i did all the steps i was'nt still able to access the web-page.
After i uninstalled open vmware tools (sudo apt-get purge --auto-remove open-vm-tools) and after an reboot i can access the web-page now.
I think it was the virtual network adaptor in the KMS VM.

Is it possible to add a executable script to the appliace to make the static ip config more easy?

**Comment by gwojcieszczuk** [comment]
i will come up with something and post it on this project page.
